### DISCUSSION TITLE
v1.36.0

### DISCUSSION THREAD

**Discussion by Koenkk** [Koenkk/zigbee2mqtt#21639] [General]
Feel free to discuss the [v1.36.0](https://github.com/Koenkk/zigbee2mqtt/releases/tag/1.36.0) release here!

**Comment by hankfic** [comment]
Getting a version mismatch on fresh install with latest Silabs firmware flash using Sonoff Dongle-E on both stable and edge. Any ideas how to resolve?

> Error: NCP EZSP protocol version of 12 does not match Host version 13

**Reply by Nerivec** [reply]
If you want to use `ember`, you have to update firmware to 7.4.x. Your adapter is reporting EZSP v12, so it is only 7.3.x.
The web flasher currently only gives you 7.3.x, if you don't supply the file yourself.
https://github.com/darkxst/silabs-firmware-builder/blob/main/firmware_builds/zbdonglee/ncp-uart-hw-v7.4.1.0-zbdonglee-115200.gbl

See https://github.com/Koenkk/zigbee2mqtt/discussions/21462

**Reply by hankfic** [reply]
Thanks, that did the trick 👍🏻

**Comment by hankfic** [comment]
Having an issue where my entire network is lost after running well for 6 hours or so. Once it gets in this state, running the Map shows all devices but no connected routes to Coordinator or anything. An uninstall and reinstall of Z2M fixes it. I haven't had this issue with prior versions. This is happening on both stable and Edge using the ember adapter.

**Reply by Nerivec** [reply]
Can you give some more details on your setup (your machine, how is Z2M running, HA addon, bare, etc)?
Do you have access to prior logs, so we can check what was going on when it lost track of devices?

**Reply by hankfic** [reply]
Sure. I'm running Z2M installed from the native Add-On store on Home Assistant OS 12.0 on VirtualBox. I haven't found anything useful in the logs when the problem occurs, looks like normal communications with MQTT (Mosquitto Broker) until it stops and then I restart and run the Map and see all my devices but no connected routes. I'm now running with zigbee_herdsman_debug enabled to hopefully catch more in the next occurrence. Any other logging you'd suggest?

**Reply by Nerivec** [reply]
`zigbee_herdsman_debug` will do. It is very verbose however, if you run it for a long time, expect rather large log files.

I assume you tried to leave it alone for a few minutes after restart (before doing the map), it doesn't change anything?
When Z2M is stopping, do you see any error in the line that starts with `Port closed.` (should be near the bottom)? Or around that line?

No chance the hardware the box is on has any I/O issues?

**Reply by hankfic** [reply]
Noted re: `zigbee_herdsman_debug`, thanks.

Yes, I've left alone for a few minutes after restart before doing the map, hours even, no change. Once it gets in this state, I haven't been able to recover, trying restarting from within the Z2M UI and the entire add-on but reinstalling the add-on has been the only successful fix.

I took another look through the Z2M folder logs from the past few days where it has occurred at least a couple times and nothing about `Port closed`.

I've checked to make sure the Sonoff card is still attached and showing in Settings > Hardware, but it's always still detected.

Thanks for all the ideas. Will report back with any more info I can gather from `zigbee_herdsman_debug` logs.

**Reply by Nerivec** [reply]
> nothing about Port closed

That means either Z2M didn't stop or it crashed, or it stopped but some other issue prevented it from writing the logs.
When you said `until it stops`, you mean Z2M, not MQTT, right?
If it crashed, you should clearly see an error at the end of the logs.
If it's the last one, you'd have to check the disk the virtualbox is on (and the rest of the hardware), to make sure it is not starting to fail (it would be a pretty strange coincidence that it started happening right when you decided to try `ember` but...).

**Reply by hankfic** [reply]
Since you say you're expecting the crashes to log, and they're just ending cleanly, I do have another suspicious issue then since the latest core or OS update, which I upgraded around the same time as starting using ember, that started causing random restarts of home assistant. I've found that I'm likely running out of ram as I noticed my swap getting full. I've been able to subside the restarts by keeping Node Red off which was hogging a lot of memory and wouldn't start any longer without causing a home assistant restart/crash. While it's much better, I'm still getting one or two restarts/crashes per day as I await my 2 gig upgrade coming. This is probably why Z2M isn't showing any crash logs. I'll report back after installing the new 2 gig of ram i have coming tomorrow.

**Comment by jabss** [comment]
Some Z2M upgrade (from last year) broke the support for the  _TZE200_lsanae15 ([DAC2161C power meter](https://www.zigbee2mqtt.io/devices/DAC2161C.html#tuya-dac2161c)) .
Tried this version, but unfortunately still not working...
Problem reported [here](https://github.com/Koenkk/zigbee2mqtt/issues/20500)

**Comment by TomKassel** [comment]
Since 1.36.0 it´s not possible for me to pair a new Schneider Electric Dimmers MEG5171-000.

Z2M Server: Publish 'get' 'switch_actions' to 'OG Dusche Dimmer' failed: 'Error: Cluster with name 'clipsalWiserSwitchConfigurationClusterServer' does not exist'
3.22.2024 13:15:04
ZigbeePlus
Error
Z2M Server: Publish 'get' 'indicator_mode' to 'OG Dusche Dimmer' failed: 'Error: Read 0xdc8e95fffe8074a1/3 manuSpecificSchneiderLightSwitchConfiguration(["ledIndication"], {"timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":4190,"transactionSequenceNumber":null,"writeUndiv":false}) failed (Status 'UNSUPPORTED_ATTRIBUTE')'
3.22.2024 13:15:04
ZigbeePlus
Error
Z2M Server: Exception while calling fromZigbee converter: Expected one of: 2, 0, 3, 1, got: 'undefined'}

Existing dimmers work, but constantly give error messages.
